alf/algorithms/entropy_target_algorithm.py

```diff
@@ -154,16 +157,43 @@ def train_step(self, distribution, step_type):
                 step_type=step_type,
                 loss=LossInfo(
                     loss=-entropy_for_gradient,
-                    extra=EntropyTargetLossInfo(entropy_loss=-entropy))))
+                    extra=EntropyTargetLossInfo(neg_entropy=-entropy))))
 
-    def calc_loss(self, training_info: EntropyTargetInfo):
+    def calc_loss(self, training_info: EntropyTargetInfo, valid_mask=None):
```

Comment:
never pass  argument value for `valid_mask` when `calc_loss` is called,  and is it same with `mask` in L164 ?

Reply:
valid_mask will be used in some other places.